Makefile

```diff
@@ -5,7 +5,7 @@ ORG := github.com/operator-framework
 PKG := $(ORG)/rukpak
 IMAGE_REPO=quay.io/operator-framework/plain-provisioner
 IMAGE_TAG=latest
-IMAGE=$(IMAGE_REPO):$(IMAGE_TAG)
+IMAGE?=$(IMAGE_REPO):$(IMAGE_TAG)
```

Comment:
Alternatively: update the workflow and explicitly specify the $(IMAGE) variable:

```console
$ make build-container IMAGE=quay.io/tflannag/rukpak:latest
docker build -f Dockerfile -t quay.io/tflannag/rukpak:latest .
...
```

Reply:
Yeah, I suppose, but I feel like being able to override from the environment makes composing multiple `make` invocations less verbose.